deploy/determined_deploy/gcp/cli.py

```diff
@@ -136,10 +136,16 @@ def make_up_subparser(subparsers: argparse._SubParsersAction) -> None:
         help="number of GPUs per agent instance",
     )
     optional_named.add_argument(
-        "--max-instances",
+        "--max-dynamic-agents",
         type=int,
-        default=constants.defaults.MAX_INSTANCES,
-        help="maximum number of agent instances at one time",
+        default=constants.defaults.MAX_DYNAMIC_AGENTS,
+        help="maximum number of dynamic agent instances at one time",
+    )
+    optional_named.add_argument(
```

Comment:
Since this is designed more for internal, do you think we should suppress the help message?

Reply:
yeah, made this change